src/main/webapp/WEB-INF/tags/pagetemplate.tag

```diff
@@ -29,6 +29,7 @@
             </button>
             <div class="collapse navbar-collapse justify-content-end" id="navbarNavAltMarkup">
                 <div class="navbar-nav">
+                    <a class="nav-item nav-link" href="${pageContext.request.contextPath}/">${sessionScope.user.username} ${sessionScope.email}</a>
```

Comment:
Der er ikke brug for '${sessionScope.email}'.

Jeg approver alligevel, men det skal lige ændres

Reply:
Dette er rettet